### PR TITLE
Add log filtering integration

### PR DESCRIPTION
**Pull request by Arkatufus**

### CODE REVIEW

File: README.md
Comment (Aaronontheweb) [inline README.md]: LGTM

File: README.md
Comment (Aaronontheweb) [inline README.md]: LGTM

File: src/Akka.Hosting.Tests/HostingExtensionsSpec.cs
Comment (Aaronontheweb) [inline src/Akka.Hosting.Tests/HostingExtensionsSpec.cs]: LGTM

File: src/Akka.Hosting/LoggerConfigBuilder.cs
Comment (Aaronontheweb) [inline src/Akka.Hosting/LoggerConfigBuilder.cs:101]: One thing I'm not so sure about - what happens if someone calls this and `ConfigureLogging` (or whatever it's called?) - can't we end up with one config overriding the other?

**Review by Aaronontheweb — COMMENTED** [review]
One question about the two different ways of doing this, but otherwise looks good

File: src/Akka.Hosting/LoggerConfigBuilder.cs
Comment (Arkatufus) [reply src/Akka.Hosting/LoggerConfigBuilder.cs:101]: Yes, the two implementation inside `ConfigureLoggers(builder)` and the extension method `AkkaConfigurationBuilder.WithLogFilter()` is exclusive to each other. The first being added would win because of how we retrieve `Setup` classes in `ActorSystem.Settings`

Comment (Aaronontheweb) [reply src/Akka.Hosting/LoggerConfigBuilder.cs:101]: LGTM - we decided to stick with just the `LogConfigBuilder`.

File: README.md
Comment (Aaronontheweb) [inline README.md]: LGTM - glad we consolidated this.

**Review by Aaronontheweb — APPROVED** [review]
LGTM (once API approvals are in)

**Pull request merged**
